scripts/db-heal-test.py

```diff
@@ -0,0 +1,231 @@
+import os
+import subprocess
+import tempfile
+import time
+import shutil
+from pathlib import Path
+import sys
+import flag
+
+# --- Dynamic variables --- #
+aida_path_flag = flag.string("aida", "", "Path to Aida root.")
+aida_db_path_flag = flag.string("aida_db", "", "Path to AidaDB.")
+tmp_path_flag = flag.string("tmp", "", "Path to tmp dir.")
+
+number_of_iterations_flag = flag.int("iter", 1000, "Number of iterations.")
+window_flag = flag.int("window", 120, "How often will program get terminated (in seconds).")
+checkpoint_granularity_flag = flag.int("cp_granularity", 2000, "How often will Carmen create checkpoints.")
+flag.parse()
```

Comment:
These default parameters look very relaxed. Only one iteration every 2 minutes is not a high frequency.

Also, 1000 iterations with 2 minutes sync periods take at least 1.3 days.

Why not checkpoint after each block, or after each 10 blocks, and kill after 5 seconds?

Reply:
I normally run this script with checkpoint every block and kill after 1s. These defaults are archaic from earlier versions. Thanks for noticing.